src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -77,7 +77,7 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
          */
         fun checkLinkingFinished( importedGeneralPersonFqns: Set<String>): Boolean {
             val finishedEntitySets = realtimeLinkingApi.linkingFinishedEntitySets
-            val finished = importedGeneralPersonFqns.all { finishedEntitySets.contains(entitySetsApi.getEntitySetId(it)) }
+            val finished = importedGeneralPersonFqns.all { finishedEntitySets.contains(edmApi.getEntitySetId(it)) }
```

Comment:
shouldn't this be entitySetsApi ?

Reply:
Same here, the wrong version got merged. Pushed the fix